src/frontend/src/lib/components/ui/Badge.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+	export let styleClass: string | undefined = undefined;
+
+	const variantClassNames = {
+		default: 'border border-light-grey bg-dust/30',
+		info: 'bg-light-blue text-blue-ribbon',
+		error: 'bg-rusty-red/20 text-rusty-red',
+		warning: 'bg-american-orange/20 text-american-orange',
+		success: 'bg-green-crayola/20 text-green-crayola',
+		outline: 'border border-light-grey bg-off-white'
+	};
+
+	export let variant: keyof typeof variantClassNames = 'default';
+</script>
+
+<span
```

Comment:
Forgot to ask, one last question: ensuring text are trimmed if too long is repsonsability of consumer of the badge?

Reply:
Good point. I could add a text-ellipsis but would expect the consumer to define the width of the badge?

Reply:
Yeah I'm not sure neither, that's why I asked ;)

Maybe have a try with your current feature, what happen if a `tag` is super long?

Note: I'll approve the PR that's something we can improve in another PR.